assets/scripts/lobby/options.js

```diff
@@ -275,6 +275,33 @@ var userOptions = {
         }
     },
 
+   optionProposedTeamIcon: {
```

Comment:
should be optionDisplayProposedTeamIcon

Reply:
Accepted. Will be resolved in the next commit